### PR TITLE
Feature use simplified jbeams if available

### PR DESCRIPTION
**Pull request by 20dka**

### CODE REVIEW

File: lua/ge/extensions/MPVehicleGE.lua [hunk 1]
```diff
@@ -37,6 +37,17 @@ local roleToInfo = {
 	['MDEV']	= { backcolor = ColorI(194, 055, 055, 127), tag = " [BeamMP Dev]", shorttag = " [Dev]" }
 }
 
+local simplified_vehicles = {
+    etk800 = "simple_traffic_etk800",
+    etkc = "simple_traffic_etkc",
+    etki = "simple_traffic_etki",
+    legran = "simple_traffic_legran",
+    midsize = "simple_traffic_midsize",
+    pessima = "simple_traffic_pessima_parked",
```
Comment (Mack29446) [inline lua/ge/extensions/MPVehicleGE.lua]: Please change this to a non-parked version of the pessima

**Review by Mack29446 — CHANGES_REQUESTED** [review]

**Comment by Olrosse** [comment]
Unless i missed it it seems like the covet is missing from the list, i think that might be one of the cars with the name simplified instead of simple

**Comment by 20dka** [comment]
> Unless i missed it it seems like the covet is missing from the list, i think that might be one of the cars with the name simplified instead of simple

fixed!

**Review by Mack29446 — APPROVED** [review]
Thank you for making the relevant changes. I have tested this and it works.

**Review by snepsnepsnep — APPROVED** [review]

**Pull request merged**